packages/i18n/src/Loader.js

```diff
@@ -201,12 +214,14 @@ EnyoLoader.prototype.loadFiles = function (paths, sync, params, callback) {
 					}
 				};
 
-				if (this.isAvailable(_root, path)) {
-					get(this._pathjoin(_root, path), handler);
+				this.loadManifestsSync(_root);
+
+				if (this.isAvailable(_root, path, true)) {
+					getSync(this._pathjoin(_root, path), handler);
 				}
 
-				if (!found && this.isAvailable(locdata, path)) {
-					get(this._pathjoin(locdata, path), handler);
+				if (!found && this.isAvailable(locdata, path, true)) {
```

Comment:
what does this thrid parameter do?

Reply:
I think at one point it was a `sync` flag but now it's just cruft. :/ 🗑